docs/content/guide/dashboard.md

```diff
@@ -144,41 +144,86 @@ export default function(req, res) {
       displayedColumns: ["position", "name", "weight", "symbol"]
     }
   };
-  response.send([
+  res.send([
+    linedata,
     bubbledata,
     scatterdata,
-    linedata,
     piedata,
     radardata,
     doughnutdata,
     polarAreadata,
     bardata,
-    tabledata
-  ]);
+    tabledata]
+  );
 }
 ```
 
 And initialize dashboards on another function:
 
-```typescript
+```javascript
 // Add `@spica-devkit/dashboard` to your dependencies and import it
 import * as Dashboard from "@spica-devkit/dashboard";
-export default async function () {
+export default async function (req,res) {
   // Create an API key and initialize dashboards with it
-  Dashboard.initialize(<APIKEY>);
+
+  Dashboard.initialize("1i7z1kcoxehci");
+  await Dashboard.remove('unique_key');
   let dashboard = await Dashboard.create({
     key: "unique_key",
     name: "Line Chart", // This name will be shown on Spica Client
     icon: "none",
     components: [
+        {
+        type: "bar",
+        // Url of your dashboard function
+        url: "https://master.spicaengine.com/api/fn-execute/line",
+        key: "bardata",
+      }, 
+      {
+        type: "table",
+        // Url of your dashobard function
```

Comment:
Maybe we should not explicitly use the functions as the data source so that the user doesn't assume that data source URLs have to be a function with HTTP trigger. 

For instance, we could fetch data from a weather forecast API?